app/src/main/AndroidManifest.xml

```diff
@@ -60,7 +60,8 @@
 
         <activity android:name=".splashScreen.SplashScreenActivity"
             android:theme="@style/SplashTheme"
-            android:screenOrientation="portrait">
+            android:screenOrientation="portrait"
+            android:exported="true">
```

Comment:
@NikolaChachev When I delete that line, I get this error message when building: 
> Manifest merger failed : android:exported needs to be explicitly specified for element <activity#com.kidssaveocean.fatechanger.splashScreen.SplashScreenActivity>. Apps targeting Android 12 and higher are required to specify an explicit value for `android:exported` when the corresponding component has an intent filter defined. See https://developer.android.com/guide/topics/manifest/activity-element#exported for details.

When I started to research the bug, I found the next point: 
> Starting from Android 12 (API level 31), you must explicitly declare the `android:exported` attribute for activities, services, and broadcast receivers that have intent filters. The system throws an exception if an activity, service, or broadcast receiver that has intent filters does not have this attribute explicitly declared in the app's manifest file.

So, I probably just need to set it `false`?

Reply:
I think, I'll merge this PR and you can fix things later.